components/network/samba/samba.p5m

```diff
@@ -89,7 +89,7 @@ link path=usr/bin/nmblookup target=../lib/samba/bin/$(MACH64)/nmblookup
 link path=usr/bin/ntlm_auth target=../lib/samba/bin/$(MACH64)/ntlm_auth
 link path=usr/bin/oLschema2ldif target=../lib/samba/bin/$(MACH64)/oLschema2ldif
 link path=usr/bin/pdbedit target=../lib/samba/bin/$(MACH64)/pdbedit
-link path=usr/bin/pidl target=../lib/samba/bin/$(MACH64)/pidl
+#link path=usr/bin/pidl target=../lib/samba/bin/$(MACH64)/pidl
```

Comment:
We don't provide `Parse:Yapp` Perl module, so no `pidl`. I think you can remove the commented link.

Reply:
I just commented that out, because there is a patch pending at upstream which should re-add pidl as an install target in the near future. If it is fine let is just commented out. I will add a comment on that in the line above.